rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/100_avg_metric.yml

```diff
@@ -0,0 +1,176 @@
+setup:
+  - do:
+      indices.create:
+          index: test_1
+          body:
+            settings:
+              number_of_replicas: 0
+            mappings:
+              test:
+                properties:
+                  int_field:
+                     type : integer
+                  double_field:
+                     type : double
+                  string_field:
+                     type: keyword
+  - do:
+      cluster.health:
+        wait_for_status: green
+
+  - do:
+       index:
```

Comment:
I'd do these as a `_bulk` instead just as a matter of habit. It'll be marginally faster and shorter to read.